lib/instruments.js

```diff
@@ -149,8 +150,8 @@ class Instruments {
     this.proc.stderr.setEncoding('utf8');
     let actOnStderr = (output) => {
       if (this.launchTimeout.afterSimLaunch && output && output.match(/CLTilesManagerClient: initialize/)) {
-        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async () => {
-          await killAllInstruments();
+        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async (done) => {
```

Comment:
No, the solution looks good. I'm just worried that the `done` stuff will be confusing. I mean, it's confusing to me! :) It seems to be used when there is success and when there is an error (like no `proc`) in `killInstruments`.